### PR TITLE
#1326 tooltip in profile nuxt

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [ ] PR closes #1404 
- [ ] <brief_description_of_what_I've_added>

### Had issue bounty label ?
- [ ] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=<My_Kusama_Address>)

### Community participation
- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

File: pages/rmrk/u/_id.vue [hunk 1]
```diff
@@ -42,6 +42,105 @@
           <DonationButton :address="id" />
         </Sharing>
       </div>
+
+      <b-tabs
+        :class="{ 'invisible-tab': sharingVisible }"
+        class="collection-tabs"
```
Comment (roiLeo) [inline pages/rmrk/u/_id.vue:48]:
```suggestion
```

File: pages/rmrk/u/_id.vue [hunk 2]
```diff
@@ -42,6 +42,105 @@
...
+            @change="totalCollected = $event"
+            :account="id"
+            showSearchBar
+          />
+        </b-tab-item>
+
+        <!-- <b-tab-item label="Packs" value="pack">
+          <span>TODO: Reintroduce</span>
+          <GalleryCardList :items="packs" type="packDetail" link="rmrk/pack" />
+        </b-tab-item> -->
```
Comment (roiLeo) [inline pages/rmrk/u/_id.vue:142]:
```suggestion
```

**Review by roiLeo — CHANGES_REQUESTED** [review]
- Duplicate Tabs in profile

**Comment by yangwao** [comment]
yeah works for me original feature, 

image

but behaves broken, might not be related to this commit, might be 

image

**Comment by roiLeo** [comment]
Hey! @prachi00 It seems that we still have the duplicated tabs here

**Comment by prachi00** [comment]
> Hey! @prachi00 It seems that we still have the duplicated tabs here

@roiLeo where? I am unable to see it

**Comment by roiLeo** [comment]
> > Hey! @prachi00 It seems that we still have the duplicated tabs here
> 
> @roiLeo where? I am unable to see it

See preview:
https://deploy-preview-1484--koda-nuxt.netlify.app/rmrk/u/HCWd8fTKDejDe8nq2qJXzF3U9J3DZx3SQXUNmwLxC1sLrso
Something went wrong.

**Comment by prachi00** [comment]
> > > Hey! @prachi00 It seems that we still have the duplicated tabs here
> > 
> > 
> > @roiLeo where? I am unable to see it
> 
> See preview: https://deploy-preview-1484--koda-nuxt.netlify.app/rmrk/u/HCWd8fTKDejDe8nq2qJXzF3U9J3DZx3SQXUNmwLxC1sLrso Something went wrong.

Have fixed this

File: pages/rmrk/u/_id.vue [hunk 3]
```diff
@@ -44,72 +44,98 @@
...
-      </b-tab-item>
-      <b-tab-item
-        :label="`Collections - ${totalCollections}`"
-        value="collection"
+      <b-tabs
+        :class="{ 'invisible-tab': sharingVisible }"
+        v-model="activeTab"
+        destroy-on-hide
+        expanded
+        size="is-medium"
```
Comment (roiLeo) [inline pages/rmrk/u/_id.vue:52]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]
lgtm, one small change + indentation fix on `<b-tabs>`

**Comment by yangwao** [comment]
Perfect, I’ve sent the payout 
https://kusama.subscan.io/extrinsic/0x20b6b4092c333f579ea953c5b322924a4e1be8f7950b3abbd2185d1f1d15476d

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**